src/network/postgres_protocol_handler.cpp

```diff
@@ -326,6 +328,33 @@ ProcessResult PostgresProtocolHandler::ExecQueryMessage(
       ExecQueryMessageGetResult(status);
       return ProcessResult::COMPLETE;
     };
+    case QueryType::QUERY_EXPLAIN: {
+      parser::ExplainStatement *explain_stmt =
+          static_cast<parser::ExplainStatement *>(sql_stmt.get());
+      std::unique_ptr<parser::SQLStatementList> unnamed_sql_stmt_list(new parser::SQLStatementList());
+      unnamed_sql_stmt_list->PassInStatement(
+          std::move(explain_stmt->real_sql_stmt));
+      auto stmt = traffic_cop_->PrepareStatement(
+          "explain", query, std::move(unnamed_sql_stmt_list), error_message);
+      ResultType status = ResultType::UNKNOWN;
+      if (stmt) {
```

Comment:
`stmt != nullptr`

Reply:
No problems in changing :) that but just curious, is that some sort of style we want to follow here? or?

Reply:
Try to be explicit, especially because `auto` is hiding the fact that it's a pointer.